admin-dev/themes/new-theme/js/pages/import-data/EntityFieldsValidator.js

```diff
@@ -0,0 +1,88 @@
+/**
+ * 2007-2018 PrestaShop.
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.txt.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to http://www.prestashop.com for more information.
+ *
+ * @author    PrestaShop SA <contact@prestashop.com>
+ * @copyright 2007-2018 PrestaShop SA
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ * International Registered Trademark & Property of PrestaShop SA
+ */
+
+const $ = window.$;
+
+export default class EntityFieldsValidator {
+  /**
+   * Validates entity fields
+   *
+   * @returns {boolean}
+   */
+  static validate() {
+    $('.js-validation-error').addClass('d-none');
+
+    return this._checkDuplicateSelectedValues() && this._checkRequiredFields();
+  }
+
+  /**
+   * Checks if there are no duplicate selected values.
+   *
+   * @returns {boolean}
+   * @private
+   */
+  static _checkDuplicateSelectedValues() {
+    let uniqueFields = [];
+    let valid = true;
+
+    $('.js-entity-field').find('select').each(function () {
```

Comment:
̀.js-entity-field select`, no need to waste memory with find

Reply:
Isn't it much slower this way?

Reply:
Nope it's like using native `querySelector`